src/views/ContributorRewards/ContributorRewards.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import Page from '../../components/Page'
+import styled from 'styled-components'
+import { Container, Spacer } from 'react-neu'
+import MonthlyClaim from './components/MonthlyClaim'
```

Comment:
order imports + use the shorthand for Page here
`import Page from 'components/Page'`